neet/synchronous.py

```diff
@@ -82,7 +82,7 @@ def landscape(self, index=None, pin=None, values=None):
         self.__values = values
 
         self.__expounded = False
-        self.__transition_graph = None
+        # self.__transition_graph = None
```

Comment:
I went back and forth on this, and honestly I just changed it to keep the functionality consistent with how the graphing works on the network itself. And in that case, we don't want to cache the network because you might be creating thousands of random networks. I also figured that this isn't going to be a limiting part of someone's pipeline (which is probably naive). Should I change it back?

Reply:
What are your thoughts on having the networkx graph as a cached value for the network? is there a reason we don't want to do that?

Reply:
Well, here's my thinking. If you are analyzing a single network, most of the time it is best to cache as much as possible. That way, repeated calls to a given property, like `attractors`, aren't terribly expensive. On the other hand, if you are analyzing networks _en masse_, like the random ensembles, you are right that the caching will lead to memory overflows if you are not careful. The question comes down to which use case is most common. My hunch is that the former is what *most* people will do. Our research program is a bit odd.

With that in mind, I'd err on the side of caching. Let say you are doing _en masse_ analyses. Then you can do something like this:
```
ensemble = [random_logic(s_pombe) for _ in range(1000)]
for i, net in enumerate(ensemble):
    net.draw_landscape()
    net.clear_landscape()
```
Though, we'll have to modify `clear_landscape()` to also reset the `transition_graph = None`. You'd have to do this anyway if you are, for example, computing the attractors.

What do you think?

Reply:
Do we need to add a way to clear the `transition_graph`? That functionality is not in my latest pull request.